osu.Game/Overlays/Notifications/Notification.cs

```diff
@@ -161,24 +160,27 @@ public virtual void Close()
             Expire();
         }
 
-        private class CloseButton : OsuClickableContainer
+        public class NotificationCloseButton : OsuClickableContainer
         {
             private Color4 hoverColour;
 
-            public CloseButton()
+            public NotificationCloseButton()
             {
                 Colour = OsuColour.Gray(0.2f);
-                AutoSizeAxes = Axes.Both;
 
-                Children = new[]
+                SpriteIcon icon;
+                Child = icon = new SpriteIcon
                 {
-                    new SpriteIcon
-                    {
-                        Anchor = Anchor.Centre,
-                        Origin = Anchor.Centre,
-                        Icon = FontAwesome.Solid.TimesCircle,
-                        Size = new Vector2(20),
-                    }
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Icon = FontAwesome.Solid.TimesCircle,
+                    RelativeSizeAxes = Axes.Both,
+                };
+
+                Enabled.ValueChanged += e =>
+                {
+                    // Hide the button internally to avoid conflicting with notification hovering logic.
+                    icon.FadeTo(e.NewValue ? 1 : 0, 120);
```

Comment:
This doesn't work. The button still has `HoverClickSounds` on it, so it's making sounds if you click where it should be while it's invisible.